resqpy/model.py

```diff
@@ -2913,14 +2913,15 @@ def wells(self):
       """
       import resqpy.organize  # Imported here to avoid circular imports
```

Comment:
for the versions of python we are using, circular imports should not cause a problem; however, it can still be good to have delayed imports so that the time is only gobbled up if the module is actually needed